scripts/release/mule/deploy/deb/create_and_push

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# This script is doing the following:
+# 1. Copying the new packages from staging to the packages cache.
+# 2. Syncing from the packages cache to the local container.
+# 3. Aptly operations:
+#       i. Add all the packages to the correct $REPO. This includes the newly-synced packages.
+#      ii. Create the new snapshot that contains the new packages and the previous ones for
+#          the channel.
+#     iii. Switch out the old snapshot for the new one. Note that in the "publish" command that
+#          the $CHANNEL is the distro.
+# 4. Copy the new packages from staging to `algorand-dev-deb-repo` so that the releases page can
+#    be generated.
+
+CHANNEL=${CHANNEL:-stable}
+REPO=${REPO:-algorand}
+
+usage() {
+    echo "Usage: -e CHANNEL=\$CHANNEL -e VERSION=\$VERSION -e REPO=\$REPO -e SNAPSHOT=\$SNAPSHOT $0 "
+    exit "$1"
+}
+
+if [ "$HELP" = true ]
+then
+    echo ------------------------------
+    echo "default values:"
+    echo + CHANNEL=stable
+    echo + REPO=algorand
+    echo + SNAPSHOT="\$CHANNEL-\$VERSION"
+    echo
+    echo "aptly repos (values for REPO):"
+    echo "+ algorand"
+    echo "+ algorand-beta"
+    echo "+ algorand-indexer"
+    echo "+ algorand-indexer-beta"
+    echo ------------------------------
+    exit 0
+fi
+
+if [ -z "$VERSION" ]
+then
+    echo "Version is a required parameter."
+    usage 1
+fi
+
+PKG_DIR="/root/packages/deb/$CHANNEL/"
+
+if [ -z "$SNAPSHOT" ]
+then
+    SNAPSHOT="$CHANNEL-$VERSION"
+fi
+
+echo -e "REPO $REPO\nSNAPSHOT $SNAPSHOT\nPKG_DIR $PKG_DIR"
+
+# It's necessary to copy packages from staging to packages/.
+SOURCE_PREFIX="algorand-staging/releases/$CHANNEL/$VERSION"
+DEST="s3://algorand-internal/packages/deb/$CHANNEL/"
+aws s3 cp "s3://$SOURCE_PREFIX/algorand_${CHANNEL}_linux-amd64_${VERSION}.deb" "$DEST"
```

Comment:
should os-arch be hardcoded here? Maybe it'd be better to grab all artifacts matching `*channel*version*` instead

Reply:
I hardcoded this because we're only supporting amd64 packages.  I can change that.

Reply:
Ok, I remember more now :)

Since `cp` doesn't support globs, I hardcoded this rather than having to use project scripts to determine the OS and ARCH_TYPE so we're not having to either copy those project scripts to this dir with the dockerfile or else change the context to the entire project.

I have some ideas to better this, but for now this is ok since our repo only supports amd64.

Reply:
```
aws s3 cp tmp/node_pkgs s3://algorand-internal/channel/${CHANNEL}/$(FULLBUILDNUMBER) --recursive --exclude "*" --include "*${CHANNEL}*$(FULLBUILDNUMBER)*"
```

I used this weird include/exclude stuff in the aws cp command to do that for copying to s3, i wonder if it works the same in reverse. We can try that stuff later though 👍